File: app/src/main/java/com/drivingschoolrwandaapp/models/response/UserSubscriptionResponse.java
package com.drivingschoolrwandaapp.models.response;

import com.drivingschoolrwandaapp.models.entities.UserSubscription;
import com.google.gson.annotations.SerializedName;

public class UserSubscriptionResponse {

    @SerializedName("success")
    private boolean success;

    @SerializedName("data")
    private UserSubscription data;

    public boolean isSuccess() {
        return success;
    }

    public UserSubscription getData() {
        return data;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/models/response/UserSubscriptionResponse.kt
package com.drivingschoolrwandaapp.models.response

import com.drivingschoolrwandaapp.models.entities.UserSubscription
import com.google.gson.annotations.SerializedName

data class UserSubscriptionResponse(
    @get:JvmName("isSuccess") @SerializedName("success") var success: Boolean = false,
    @SerializedName("data") var data: UserSubscription? = null
)
